pulse2percept/utils.py

```diff
@@ -42,7 +42,7 @@ def wrap(func):
         """
         e_s = "Call to deprecated function %s." % func.__name__
         if alt_func:
-            e_s += "Use %s instead." % alt_func
+            e_s += " Use %s instead." % alt_func
```

Comment:
Sorry - I think I missed this before. Where does `alt_func` come from? 

It doesn't seem to be defined anywhere. I must be missing something